doc/source/getting-started.rst

```diff
@@ -47,4 +32,10 @@ Questions or Suggestions?
 If you have questions or suggestions, please reach out at our `issues page`_
 on GitHub.
 
+Next Step
```

Comment:
Let's take out the Questions or Suggestions section as it's in the sidebar. I would also remove Next step as a section and just leave the sentence. [The ref link has an extra `_`]